tasks/docker.py

```diff
@@ -19,7 +19,7 @@ def test(ctx):
     """
     Run docker tests
     """
-    ctx.run("python ./Dockerfiles/agent/secrets-helper/test_readsecret.py")
+    ctx.run("python3 ./Dockerfiles/agent/secrets-helper/test_readsecret.py")
```

Comment:
since we want the helper to work with both python 2 and 3 (as long as we ship v6), we should make the tests run under both versions. Not a blocker for this specific PR though.